x-pack/plugins/observability_solution/slo/public/plugin.ts

```diff
@@ -159,6 +149,16 @@ export class SloPlugin
     const kibanaVersion = this.initContext.env.packageInfo.version;
     const { ruleTypeRegistry, actionTypeRegistry } = pluginsStart.triggersActionsUi;
 
+    pluginsStart.dashboard.registerDashboardPanelPlacementSetting(
+      SLO_OVERVIEW_EMBEDDABLE_ID,
+      (serializedState: SloOverviewEmbeddableState | undefined) => {
+        if (serializedState?.showAllGroupByInstances || serializedState?.groupFilters) {
+          return { width: 24, height: 8 };
+        }
+        return { width: 12, height: 8 };
+      }
+    );
```

Comment:
I moved this to the start lifecycle since the method is provided by `DashboardStart`.